src/openassetio-python/tests/package/hostApi/test_manager.py

```diff
@@ -240,6 +241,54 @@ def test_when_entity_ref_prefix_type_invalid_then_debug_log_printed(
         )
 
 
+class Test_Manager_initizlize_capablility_check:
```

Comment:
[consider][minor] Spelling
```suggestion
class Test_Manager_initialize_capablility_check:
```